tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -386,54 +386,7 @@ def _enable_disable_volume_profiling(self):
         # should be done only once while first sync. Later the volume
         # level volume_profiling_state should be set based on individual
         # volume level values
-        _cnc = NS.tendrl.objects.ClusterNodeContext(
```

Comment:
I feel the above comment needs a change to explain that the profile enable/disable would be invoked as a separate flow now.

Reply:
Also the function name should change as its no more enabling/disabling profiling for the volumes. Rather its setting the cluster level value for profiling. Right?

Reply:
right, Will make the required change.